Bottom: d176274f78d9fd871b072c9dec403343f731196e
Top:    a1d2ee10d887ca2a9fd3048cf77e80433ae02ae7
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-08-17 11:07:45 -0400

Refresh of sgtl5000

---

diff --git a/arch/arm/boot/dts/sun7i-a20-cubietruck.dts b/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
index def4394..7fe6355 100644
--- a/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
+++ b/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
@@ -157,7 +157,7 @@
 			sgtl5000: sgtl5000@a {
 				compatible = "fsl,sgtl5000";
 				reg = <0x0a>;
-				pwms = <&pwm 0 50 0>;
+				pwms = <&pwm 0 200 0>;
 
 				#sound-dai-cells = <0>;
 				VDDA-supply = <&reg_vcc3v3>;
